### PR TITLE
Bump unattended 4.3.0

### PR DESCRIPTION
**Pull request by DFolchA**
|Related issue|
|---|
||

## Description
This PR updates the unattended installation version in branch 4.3

## Tests

- Build the package in any supported platform
  - [ ] Linux
  - [ ] Windows
  - [ ] macOS
  - [ ] Solaris
  - [ ] AIX
  - [ ] HP-UX
- [ ] Package installation
- [ ] Package upgrade
- [ ] Package downgrade
- [ ] Package remove
- [ ] Package install/remove/install
- [ ] Change added to CHANGELOG.md

- Tests for Linux RPM
  - [ ] Build the package for x86_64
  - [ ] Build the package for i386
  - [ ] Build the package for armhf
  - [ ] Build the package for aarch64
  - [ ] `%files` section is correctly updated if necessary
- Tests for Linux deb
  - [ ] Build the package for x86_64
  - [ ] Build the package for i386
  - [ ] Build the package for armhf
  - [ ] Build the package for aarch64
  - [ ] Package install/remove/install
  - [ ] Package install/purge/install
  - [ ] Check file permissions after installing the package
- Tests for macOS
  - [ ] Test the package from macOS Sierra to Mojave
- Tests for Solaris
  - [ ] Test the package on Solaris 10
  - [ ] Test the package on Solaris 11
  - [ ] Check file permissions on Solaris 11 template
- Tests for IBM AIX
  - [ ] `%files` section is correctly updated if necessary
  - [ ] Check the changes from IBM AIX 5 to 7

### CODE REVIEW

File: unattended_scripts/elastic-stack/unattended-installation/distributed/elastic-stack-installation.sh [hunk 1]
```diff
@@ -8,6 +8,13 @@
 # License (version 2) as published by the FSF - Free Software
 # Foundation.
 
+WAZUH_MAJOR="4.2"
```
Comment (okynos) [inline unattended_scripts/elastic-stack/unattended-installation/distributed/elastic-stack-installation.sh:11]: Change to 4.3

File: unattended_scripts/open-distro/unattended-installation/distributed/elastic-stack-installation.sh [hunk 2]
```diff
@@ -8,14 +8,17 @@
 # License (version 2) as published by the FSF - Free Software
 # Foundation.
 
-## Check if system is based on yum or apt-get
 char="."
 debug='> /dev/null 2>&1'
-WAZUH_VER="4.2.2"
+WAZUH_VER="4.3.0"
+WAZUH_MAJOR="4.2"
```
Comment (okynos) [inline unattended_scripts/open-distro/unattended-installation/distributed/elastic-stack-installation.sh:14]: Change to 4.3

File: unattended_scripts/open-distro/unattended-installation/unattended-installation.sh [hunk 3]
```diff
@@ -12,7 +12,7 @@
 char="."
 debug='>> /var/log/wazuh-unattended-installation.log 2>&1'
 WAZUH_MAJOR="4.2"
```
Comment (okynos) [inline unattended_scripts/open-distro/unattended-installation/unattended-installation.sh:14]: Change to 4.3

**Review by okynos — CHANGES_REQUESTED** [review]
Minor changes.

File: unattended_scripts/open-distro/unattended-installation/unattended-installation.sh (hunk 3 above)
Comment (DFolchA) [reply unattended_scripts/open-distro/unattended-installation/unattended-installation.sh]: Done a600908

File: unattended_scripts/open-distro/unattended-installation/distributed/elastic-stack-installation.sh (hunk 2 above)
Comment (DFolchA) [reply unattended_scripts/open-distro/unattended-installation/distributed/elastic-stack-installation.sh]: Done a600908

File: unattended_scripts/elastic-stack/unattended-installation/distributed/elastic-stack-installation.sh (hunk 1 above)
Comment (DFolchA) [reply unattended_scripts/elastic-stack/unattended-installation/distributed/elastic-stack-installation.sh]: Done a600908

**Review by alberpilot — APPROVED** [review]
LGTM

**Review by rauldpm — CHANGES_REQUESTED** [review]
LGTM

**Review by rauldpm — APPROVED** [review]

**Review by okynos — APPROVED** [review]
LGTM.

**Pull request merged**